starter-core/src/main/java/io/micronaut/starter/feature/agora/gru/template/helloWorldGruControllerTestKotlin.rocker.raw

```diff
@@ -0,0 +1,35 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+import com.agorapulse.gru.Gru
+import com.agorapulse.gru.ResponseDefinitionBuilder
+import com.agorapulse.gru.TestDefinitionBuilder
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import jakarta.inject.Inject
+import org.junit.jupiter.api.Test
+
+@@MicronautTest
+internal class HelloWorldGruControllerTest {
+    @@Inject
+    lateinit var gru: Gru
+
+    @@Test
+    @@Throws(Throwable::class)
+    fun testGet() {
+        gru.verify { test: TestDefinitionBuilder ->
+            test["/gru"]
+                .expect { response: ResponseDefinitionBuilder ->
+                    response.json(
+                        "gruIndex.json"
+                    )
+                }
+        }
+    }
```

Comment:
This can be simplified as 

```suggestion
        gru.verify { it
            .get("/gru")
            .expect { it.json("gruIndex.json") }
        }
    }
```